### DISCUSSION TITLE
Latency option

### DISCUSSION THREAD

**Discussion by dobby5** [music-assistant/support#771] [Feature requests and ideas]
Hello,

i have do some experimentings with Music Assistant. For the most of part i am very hppy with it. 

But it would be nice if a "Latnecy" option could be added to solve the asynchronicity between the devices.

I hope that is possible.

Have a nice day

**Comment by marcelveldt** [comment]
I think you have missed the various announcements about that ?
This is going to be added in the next release of MA... So stay tuned!

**Reply by dobby5** [reply]
Sorry, you're right, I missed that one.

Thanks for the really extremely quick answer :D
